notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.                    r = requests.get(url, timeout=(timeout or self.timeout))
</pre><p>use what is set in #64</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>